truenas/provider.go

```diff
@@ -35,6 +35,8 @@ func Provider() *schema.Provider {
 		ResourcesMap: map[string]*schema.Resource{
 			"truenas_cronjob": resourceTrueNASCronjob(),
 			"truenas_dataset": resourceTrueNASDataset(),
+			"truenas_share_nfs": resourceTrueNASShareNFS(),
```

Comment:
could you run `go fmt ./...`, I was thinking to add git hook for that, but never got a chance

Reply:
Whoops, forgot `./...`. Sorry I didn't catch that.